presto-tests/src/main/java/com/facebook/presto/tests/TestedFeature.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tests;
+
+import com.google.common.collect.ImmutableSet;
+
+import java.util.Set;
+
+import static java.util.Objects.requireNonNull;
+
+public enum TestedFeature
+{
+    SCAN,
```

Comment:
Maybe group features by the aggregation level, e.g:
```
* basic features (e.g: filtered aggregation, left join, create_table etc...)
* feature suites (e.g: join, aggregation, metadata, etc)
* functionality/modules (e.g: connector, execution, planner)
```
add comments what is the next features section (e.g: `//metadata, //enigne, //planner`) or `//basic features, //suites, //modules`)

Reply:
"TABLE_SCAN" (do we need this one??)

I like separation of connector features (like VIEW) and core features (like JOIN). For now, it can be at source level (just ordering with comments, as @sopel39 suggested).